Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM swaggerapi/swagger-editor
+
```

Comment:
You should add `EXPOSE` instructions with the ports that can be used with a running container.
You also may add a maintainer `LABEL` as an example `LABEL maintainer="SvenDowideit@home.org.au"`
> https://docs.docker.com/engine/reference/builder/#maintainer-deprecated